lib/browser/api/browser-window.js

```diff
@@ -82,9 +82,15 @@ BrowserWindow.prototype._init = function () {
   // Though this hack is only needed on macOS when the app is launched from
   // Finder, we still do it on all platforms in case of other bugs we don't
   // know.
-  this.webContents.once('load-url', function () {
-    this.focus()
-  })
+  this.webContents.once(
+      'load-url',
+      /**
+     * @this WebContents
```

Comment:
?

Reply:
Without declaring what `this` is, eslint rules don't think this non-class function has a `this` scope that is usable.  All the `@this` declarations in JSDoc are for this purpose 👍

Reply:
Btw, The second line of the comment block is not indented properly